OpenRA.Game/Traits/Player/PlayerResources.cs

```diff
@@ -70,6 +70,18 @@ public PlayerResources(Actor self, PlayerResourcesInfo info)
 
 		int lastNotificationTick;
 
+		public void ChangeCash(int amount)
+		{
+			if (amount >= 0)
+			{
```

Comment:
Code style nit: Braces are dropped in such single-line code blocks.

You coulda consider simplifying this into `amount >= 0 ? GiveCash(amount) : TakeCash(-amount)` as well.

Reply:
I can drop the braces, although it doesn't affect functionality in any way. 
As for the ternary, I would prefer to avoid it for several reasons.

1. Ternary operator is slightly less efficient, not enough to cause concern but still.
2. The readability kind of suffers when you use it. It is pretty obvious **what** it is. But more complex conditions can get confusing for the reader. On the other hand if-else are completely obvious from the first glance.


That being said, if it is real issue and a requirement, I will change it. No problem.

Reply:
Well, I like ternaries, but their use is optional. The braces however are still against the employed coding style - see point 7 of the list at https://github.com/OpenRA/OpenRA/wiki/Coding-Standard.